src/message/simple/index.tsx

```diff
@@ -1,56 +1,43 @@
 import React from 'react';
 import { RemoteStop, RemoteStopProps } from '../remote-stop';
-import { SetVariables, SetVariablesProps } from '../set-variables';
+import { SetVariables } from '../set-variables';
 import { TriggerMessage, TriggerMessageProps } from '../trigger-message';
-import { GetBaseReport, GetBaseReportProps } from '../get-base-report';
-import { ClearCache, ClearCacheProps } from '../clear-cache';
+import { GetBaseReport } from '../get-base-report';
+import { ClearCache } from '../clear-cache';
 import { ChangeAvailability } from '../change-availability';
-import { GetLog, GetLogProps } from '../get-log';
-import { UnlockConnector, UnlockConnectorProps } from '../unlock-connector';
-import { ResetChargingStation, ResetChargingStationProps } from '../reset';
-import { RemoteStart, RemoteStartProps } from '../remote-start';
-import { GetVariables, GetVariablesProps } from '../get-variables';
-import {
-  InstallCertificate,
-  InstallCertificateProps,
-} from '../install-certificate';
-import {
-  GetInstalledCertificateIds,
-  GetInstalledCertificateIdsProps,
-} from '../get-installed-certificate-ids';
-import {
-  SetNetworkProfile,
-  SetNetworkProfileProps,
-} from '../set-network-profile';
+import { GetLog } from '../get-log';
+import { UnlockConnector } from '../unlock-connector';
+import { ResetChargingStation } from '../reset';
+import { RemoteStart } from '../remote-start';
+import { GetVariables } from '../get-variables';
+import { InstallCertificate } from '../install-certificate';
+import { GetInstalledCertificateIds } from '../get-installed-certificate-ids';
+import { SetNetworkProfile } from '../set-network-profile';
 import {
   CertificateSigned,
   CertificateSignedProps,
 } from '../certificate-signed';
-import { CustomAction } from '../../components/custom-actions';
 import { ChargingStations } from '../../graphql/schema.types';
 import { ChargingStation } from '../../pages/charging-stations/ChargingStation';
-import { ChangeAvailabilityProps } from '../change-availability/model';
+import { CustomAction } from '@interfaces';
 
 const chargingStationActionMap: {
```

Comment:
Can we remove this file? Is this file used?

Reply:
CustomAction is been used on line 44 

export const CUSTOM_CHARGING_STATION_ACTIONS: **CustomAction**<ChargingStations>[] =